src/api/responses/txreceipt.ts

```diff
@@ -13,17 +13,31 @@ import {
 
 const txReceiptTypeName = "TxReceipt";
 
+export enum TXStatusCodes {
```

Comment:
I'd prefer `TxStatusCode` since it will then be used as `TxStatusCode.Fail` and this reads better.

Also `Tx` because our transaction types also always start like this.